test/e2e-go/restAPI/other/appsRestAPI_test.go

```diff
@@ -549,3 +550,144 @@ end:
+	byte b64 %s
+	itxn_field ClearStateProgram
+	itxn_submit
+
+	ret:
+	int 1
+	return 
+	`, b64InnerApproval, b64InnerApproval)
+
+	outerOps, err := logic.AssembleString(outerTEAL)
+	a.NoError(err)
+	outerApproval := outerOps.Program
+
+	gl := basics.StateSchema{}
+	lc := basics.StateSchema{}
+
+	// create app
+	appCreateTxn, err := testClient.MakeUnsignedApplicationCallTx(
+		0, nil, nil, nil,
+		nil, nil, transactions.NoOpOC,
+		outerApproval, clearState, gl, lc, 0,
+	)
+	a.NoError(err)
+	appCreateTxn, err = testClient.FillUnsignedTxTemplate(someAddress, 0, 0, 0, appCreateTxn)
+	a.NoError(err)
+	appCreateTxID, err := testClient.SignAndBroadcastTransaction(wh, nil, appCreateTxn)
+	a.NoError(err)
+	createConf, err := helper.WaitForTransaction(t, testClient, appCreateTxID, 30*time.Second)
+	a.NoError(err)
+
+	createdAppID := basics.AppIndex(*createConf.ApplicationIndex)
+
+	// fund app account
+	appFundTxn, err := testClient.SendPaymentFromWallet(wh, nil, someAddress, createdAppID.Address().String(), 0, 1_000_000, nil, "", 0, 0)
+	a.NoError(err)
+	appFundTxID := appFundTxn.ID()
+	_, err = helper.WaitForTransaction(t, testClient, appFundTxID.String(), 30*time.Second)
+	a.NoError(err)
+
+	// call app twice
+	appCallTxn, err := testClient.MakeUnsignedAppNoOpTx(
+		uint64(createdAppID), nil, nil, nil,
+		nil, nil,
+	)
+	appCallTxn0, err := testClient.FillUnsignedTxTemplate(someAddress, 0, 0, 0, appCallTxn)
+	appCallTxn0.Note = []byte("0")
+
+	appCallTxn1, err := testClient.FillUnsignedTxTemplate(someAddress, 0, 0, 0, appCallTxn)
+	appCallTxn1.Note = []byte("1")
+	a.NoError(err)
+
+	gid, err := testClient.GroupID([]transactions.Transaction{appCallTxn0, appCallTxn1})
+	a.NoError(err)
+	appCallTxn0.Group = gid
+	appCallTxn1.Group = gid
+
+	stxn0, err := testClient.SignTransactionWithWallet(wh, nil, appCallTxn0)
+	a.NoError(err)
+	stxn1, err := testClient.SignTransactionWithWallet(wh, nil, appCallTxn1)
+	a.NoError(err)
+
+	err = testClient.BroadcastTransactionGroup([]transactions.SignedTxn{stxn0, stxn1})
+	a.NoError(err)
+
+	callConf, err := helper.WaitForTransaction(t, testClient, stxn0.ID().String(), 30*time.Second)
+	a.NoError(err)
+
+	round := callConf.ConfirmedRound
+
+	deadDood, err := hex.DecodeString("deadd00d")
+	deadBeef, err := hex.DecodeString("deadbeef")
+
+	// get block logs
+	resp, err := testClient.BlockLogs(*round)
+	a.NoError(err)
+	l0 := resp.Logs[0]
+	l1 := resp.Logs[1]
+	l2 := resp.Logs[2]
+	l3 := resp.Logs[3]
+
+	a.Equal(uint64(createdAppID), l0.ApplicationIndex)
+	a.Equal(stxn0.ID().String(), l0.Txid)
+	a.Equal(deadDood, l0.Logs[0])
+
+	a.Equal(uint64(createdAppID)+3, l1.ApplicationIndex)
+	a.Equal(stxn0.ID().String(), l1.Txid)
+	a.Equal(deadBeef, l1.Logs[0])
+
+	a.Equal(uint64(createdAppID), l2.ApplicationIndex)
+	a.Equal(stxn1.ID().String(), l2.Txid)
+	a.Equal(deadDood, l2.Logs[0])
+
+	a.Equal(uint64(createdAppID)+5, l3.ApplicationIndex)
+	a.Equal(stxn1.ID().String(), l3.Txid)
+	a.Equal(deadBeef, l3.Logs[0])
+}
```

Comment:
I suggest this change in assertion style because it's much stronger -- if any additional fields/elements are present that shouldn't be, this will catch that.
```suggestion
    resp, err := testClient.BlockLogs(*round)
	a.NoError(err)

	expected := model.BlockLogsResponse{
		Logs: []model.AppCallLogs{
			{
				ApplicationIndex: uint64(createdAppID),
				Txid:             stxn0.ID().String(),
				Logs:             [][]byte{deadDood},
			},
			{
				ApplicationIndex: uint64(createdAppID + 3),
				Txid:             stxn0.ID().String(),
				Logs:             [][]byte{deadBeef},
			},
			{
				ApplicationIndex: uint64(createdAppID),
				Txid:             stxn1.ID().String(),
				Logs:             [][]byte{deadDood},
			},
			{
				ApplicationIndex: uint64(createdAppID + 5),
				Txid:             stxn1.ID().String(),
				Logs:             [][]byte{deadBeef},
			},
		},
	}
	a.Equal(expected, resp)
```